.gitmodules

```diff
@@ -7,3 +7,12 @@
 [submodule "third_party/mosquitto"]
 	path = third_party/mosquitto
 	url = https://github.com/eclipse/mosquitto.git
+[submodule "third_party/http-parser"]
+	path = third_party/http-parser
+	url = https://github.com/nodejs/http-parser.git
+[submodule "third_party/Unity"]
```

Comment:
The commit message should explain the considerations for duplicated copies of `Unity`.